diff --git a/plugins/category_generator.rb b/plugins/category_generator.rb
index c55d62f..5e7ead4 100644
--- a/plugins/category_generator.rb
+++ b/plugins/category_generator.rb
@@ -161,7 +161,7 @@ ERR
       when 1
         categories[0].to_s
       else
-        "#{categories[0...-1].join(', ')}, #{categories[-1]}"
+        "#{categories[0...-1].join(' ')} #{categories[-1]}"
       end
     end
 
@@ -173,7 +173,7 @@ ERR
     #
     def category_link(category)
       dir = @context.registers[:site].config['category_dir']
-      "<a class='category' href='/#{dir}/#{category.to_url}/'>#{category}</a>"
+      "<a class='category label label-primary' href='/#{dir}/#{category.to_url}/'>#{category}</a>"
     end
 
     # Outputs the post.date as formatted html, with hooks for CSS styling.
