diff --git a/plugins/code_block.rb b/plugins/code_block.rb
index c70e181..29ad53a 100644
--- a/plugins/code_block.rb
+++ b/plugins/code_block.rb
@@ -62,10 +62,10 @@ module Jekyll
       end
       if markup =~ CaptionUrlTitle
         @file = $1
-        @caption = "<figcaption><span>#{$1}</span><a href='#{$2}'>#{$3 || 'link'}</a></figcaption>"
+        @caption = "<figcaption class='panel-heading'><h3 class='panel-title'>#{$1}</h3><a href='#{$2}'>#{$3 || 'link'}</a></figcaption>"
       elsif markup =~ Caption
         @file = $1
-        @caption = "<figcaption><span>#{$1}</span></figcaption>\n"
+        @caption = "<figcaption class='panel-heading'><h3 class='panel-title'>#{$1}</h3></figcaption>\n"
       end
       if @file =~ /\S[\S\s]*\w+\.(\w+)/ && @filetype.nil?
         @filetype = $1
@@ -76,7 +76,7 @@ module Jekyll
     def render(context)
       output = super
       code = super
-      source = "<figure class='code'>"
+      source = "<figure class='code panel panel-default'>"
       source += @caption if @caption
       if @filetype
         source += " #{highlight(code, @filetype)}</figure>"
